src/components/Frame/Frame.scss

```diff
@@ -88,13 +91,30 @@ $skip-vertical-offset: rem(10px);
 
   &:focus {
     border-radius: border-radius();
-    background-color: rgba(color('white'), 0.16);
+    background-color: var(--p-override-transparent, rgba(color('white'), 0.16));
     outline: none;
   }
 
   @include frame-when-nav-displayed {
     display: none;
   }
+
+  .Navigation-globalTheming & {
+    border-radius: 50%;
+
+    &:hover {
+      background-color: var(--p-overlay);
+    }
+
+    &:focus {
+      @include focus-ring;
+      position: absolute;
```

Comment:
Is it frustrating that the focus ring sets position relative on its parent? Or is this a fringe case?

Reply:
I haven't dug into the focus-ring mixin enough to be sure